### ISSUE TITLE
Incomplete cases for function_importNames_csv.R

### ISSUE THREAD

**Issue by FlicAnderson** [FlicAnderson/Scriptbox#8]
Need to complete case behaviours for function_importNames_csv.R in the event of different combinations of column arrangement for sp, ssp & authorities.

Example: 
column index for sp: 4, 
column index for ssp: 4 (ssp in same column as sp)
column index for authorities: 5

This is not catered for & no useful error thrown.

Neither if ALL columns are different, or species is in different column. 

Basically, complete the behaviours to allow script_latinNamesMatcher.R to run on all combinations.
